<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientClass</class>
 <widget class="QMainWindow" name="clientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="Amain">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>47</y>
      <width>651</width>
      <height>321</height>
     </rect>
    </property>
    <widget class="QWidget" name="tabWidgetPage1" native="true">
     <attribute name="title">
      <string/>
     </attribute>
     <layout class="QGridLayout" name="AMain">
      <item row="6" column="2">
       <layout class="QGridLayout" name="Bmain">
        <item row="1" column="0">
         <widget class="QLabel" name="Namelabel">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Name: </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="nameLINE">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="idLINE">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="Pricelabel">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Price: </string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="priceLINE">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="quantityLINE">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="Quantitylabel">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Quantity: </string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="IDlabel">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>ID: </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2" rowspan="5">
       <widget class="QTextBrowser" name="responseTXT"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="B2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>2. Удалить продукт</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="B5">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>5. Просмотреть список всех продуктов</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="B1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>1. Добавить продукт</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="B3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>3. Обновить информацию о продукте</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="B4">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>4. Просмотреть информацию о продукте</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="B6">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>6. Выход</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>923</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="client.qrc"/>
 </resources>
 <connections/>
</ui>
